fractal_server/app/runner/_slurm/remote.py

```diff
@@ -87,8 +87,8 @@ def worker(
         tb = tb.tb_next
         exc_proxy = ExceptionProxy(
             typ,
-            traceback.format_exception(typ, value, tb),
-            e.args,
+            "".join(traceback.format_exception(typ, value, tb)),
+            *e.args,
```

Comment:
@mfranzon I think that this line change (from `e.args` to `*e.args`) was actually crucial, to avoid propagating an actual tuple to the downstream exceptions.